src/mono/System.Private.CoreLib/src/System/TypeLoadException.Mono.cs

```diff
@@ -13,6 +13,18 @@ internal TypeLoadException(string className, string assemblyName)
             _assemblyName = assemblyName;
         }
 
+        // Because the Mono runtime has a dependency to a (string, string) constructor overload,
+        // we add a dummy parameter with a default value to minimize native code changes.
+        // In order to use this overload, callers should either pass three arguments, or specify
+        // one of the parameters by name.
```

Comment:
```suggestion
        // Because the Mono runtime has a dependency on a (string, string) constructors of exception types,
        // we add a dummy parameter with a default value.
        // In order to use this overload, callers should specify
        // the typeName parameter by name.
```

Reply:
Done.